apps/language_server/lib/language_server/server.ex

```diff
@@ -729,6 +730,17 @@ defmodule ElixirLS.LanguageServer.Server do
      end, state}
   end
 
+  defp handle_request(folding_range_req(_id, uri), state) do
+    case state.source_files[uri] do
```

Comment:
use `source_file = Server.get_source_file(state, uri)` as in other handlers

Reply:
Fixed by: 4458bdf